docs/policy/container-release.md

```diff
@@ -22,16 +22,53 @@ EPEL, and [OSG](../policy/software-release.md#yum-repositories) Yum repositories
 Tags
 ----
 
-OSG Software container images will be built at least weekly and tagged with one of the following:
+OSG Software container images will be built at least weekly and tagged with the following format:
 
-| Tag                            | Description                                                                                                              |
-|--------------------------------|--------------------------------------------------------------------------------------------------------------------------|
-| `testing`                      | For the latest fixes and features. Built with RPMs from the `osg-testing` and/or `osg-upcoming-testing` Yum repositories |
-| `release`                      | For production use. Built with RPMs from the `osg-release` and/or `osg-upcoming-release` Yum repositories                |
-| `rc.testing.*`, `rc.release.*` | Release candidate images; see the [Validation section below](#validation) for details                                    |
+```
+<SERIES>-<REPO>[-<TIME>]
```

Comment:
Eh. Six of one, half a dozen of another.
I don't want it to look like what we call our 3.6 version numbers (3.6.220113) otherwise it will confuse people.

Reply:
Fair point!